pytorch_lightning/accelerators/ddp_accelerator.py

```diff
@@ -133,11 +133,13 @@ def _call_children_scripts(self):
                 del env_copy['PL_GLOBAL_SEED']
 
             # start process
-            # if hydra is available and initialized, make sure to set the cwd correctly
+            # if hydra is available and initialized, make sure to set the original cwd correctly
+            # and pass current cwd for ddp processes (which hydra has overridden)
             cwd: Optional[str] = None
             if HYDRA_AVAILABLE:
                 if HydraConfig.initialized():
                     cwd = get_original_cwd()
+                    command += [f'hydra.run.dir={os.getcwd()}']
```

Comment:
this dir or rather the trainer default dir?

Reply:
The Hydra directory I think is the right one to set, internally hydra has set this directory where the trainer directory is now inside (unless absolute I think), which fixes it as Omry suggested!